<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color:black;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>480</width>
      <height>320</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{
background-color:black;
border-color:black;
}
QTabWidget{
background-color:black;
}
QTabBar::tab{
color:white;
background-color:black;
}
QTabBar{
background-color:black;
color:black;
}</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::West</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>General</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>0</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;
}</string>
      </property>
      <property name="text">
       <string>Corner Pressures</string>
      </property>
     </widget>
     <widget class="QLabel" name="front_left">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>68</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;
}</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="back_left">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>68</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;
}</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="front_right">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>68</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;
}</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="back_right">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>140</y>
        <width>68</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;
}</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="tank">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>68</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;
}</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="preset1">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: grey;
color:white;
}
</string>
      </property>
      <property name="text">
       <string>Preset 1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="preset2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>100</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: grey;
color:white;
}

</string>
      </property>
      <property name="text">
       <string>Preset 2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="preset3">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>180</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: grey;
color:white;
}

</string>
      </property>
      <property name="text">
       <string>Preset 3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AirOut">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>241</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: red;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
color:white;
}
QPushButton:pressed {
    background-color: rgb(224, 0, 0);
    border-style: inset;
}</string>
      </property>
      <property name="text">
       <string>AIR OUT BOIIIII</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Preset Adjust</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;
}</string>
      </property>
      <property name="text">
       <string>Preset 1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;
}</string>
      </property>
      <property name="text">
       <string>Preset 2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;
}</string>
      </property>
      <property name="text">
       <string>Preset 3</string>
      </property>
     </widget>
     <widget class="QSlider" name="Preset1Mod">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>371</width>
        <height>26</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="Preset2Mod">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>371</width>
        <height>26</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="PresetMod3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>200</y>
        <width>371</width>
        <height>26</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="Preset1Mod_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>371</width>
        <height>26</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="Preset2Mod_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>371</width>
        <height>26</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="PresetMod3_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>230</y>
        <width>371</width>
        <height>26</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="P1Flabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;
}</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="P1Blabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;
}</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="P2Flabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;
}</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="P2Blabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;
}</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="P3Flabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;
}</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="P3Blabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;
}</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Manual</string>
     </attribute>
     <widget class="QPushButton" name="FL_UP">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: white;
color:black;
}
</string>
      </property>
      <property name="text">
       <string>Up</string>
      </property>
     </widget>
     <widget class="QPushButton" name="FL_DOWN">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: white;
color:black;
}
</string>
      </property>
      <property name="text">
       <string>Down</string>
      </property>
     </widget>
     <widget class="QPushButton" name="FR_UP">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>20</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: white;
color:black;
}
</string>
      </property>
      <property name="text">
       <string>Up</string>
      </property>
     </widget>
     <widget class="QPushButton" name="FR_DOWN">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>50</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: white;
color:black;
}
</string>
      </property>
      <property name="text">
       <string>Down</string>
      </property>
     </widget>
     <widget class="QPushButton" name="RL_UP">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: white;
color:black;
}

</string>
      </property>
      <property name="text">
       <string>Up</string>
      </property>
     </widget>
     <widget class="QPushButton" name="RL_DOWN">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: white;
color:black;
}
</string>
      </property>
      <property name="text">
       <string>Down</string>
      </property>
     </widget>
     <widget class="QPushButton" name="RR_UP">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>110</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: white;
color:black;
}
</string>
      </property>
      <property name="text">
       <string>Up</string>
      </property>
     </widget>
     <widget class="QPushButton" name="RR_DOWN">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>140</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: white;
color:black;
}
</string>
      </property>
      <property name="text">
       <string>Down</string>
      </property>
     </widget>
     <widget class="QLabel" name="FL_MAN">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>68</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;
}</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="FR_MAN">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>20</y>
        <width>68</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;
}
</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="RR_MAN">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>110</y>
        <width>68</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;
}
</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="RL_MAN">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>110</y>
        <width>68</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
color:white;
}</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ON">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>190</y>
        <width>301</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: white;
color:black;
    
}
</string>
      </property>
      <property name="text">
       <string>On</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
